### PR TITLE
Closes #1 Create initial index.html

### LINKED ISSUE #1: Create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (AnnetteDr) [inline index.html]: Please add a new line in the end

Comment (TamaraKovalchuk) [inline index.html]: please add a new line

Comment (carbonid1) [inline index.html]: where is the new line?

Comment (koldovsky) [reply index.html]: Fixed

Comment (koldovsky) [reply index.html]: Done 😀

Comment (koldovsky) [reply index.html]: Fixed 👻

**Pull request merged**